docs/Configuring a Harmony service.ipynb

```diff
@@ -0,0 +1,577 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Configuring a Harmony Service\n",
+    "\n",
+    "This notebook will show the steps required to configure a [Harmony](https://harmony.earthdata.nasa.gov/) service, covering the following points:\n",
+    "\n",
+    "* Initial configuration of a new service in Harmony.\n",
+    "* Associating a collection with an existing service.\n",
+    "* Enabling service discovery via Earthdata Search Client ([EDSC](https://search.earthdata.nasa.gov/search)).\n",
+    "* Associating variables with an existing collection.\n",
+    "\n",
+    "\n",
+    "The following requirements are assumed to be already fulfilled:\n",
+    "\n",
+    "* A Docker image, containing a service that is wrapped in a `HarmonyAdapter` instance, exists in a place that can be accessed by Harmony.\n",
+    "* A collection, containing granules, has been ingested and has associated UMM-C and UMM-G records.\n",
+    "* Write access to the CMR provider containing the collection to be associated with the new Harmony service.\n",
+    "* Access to the [NASA harmony repository](https://github.com/nasa/harmony), including the ability to push branches to the remote repository, and to open pull requests (PRs).\n",
+    "\n",
+    "The cell below will import the packages required for this notebook - it will need to be run ahead of most of the cells below that make requests against the CMR and UVG APIs."
```

Comment:
```suggestion
    "The cell below will import the packages required for this notebook - it will need to be run ahead of most of the cells below that make requests against the CMR and the UMM-Var Generator (UVG) APIs."
```